contracts/front-end-helpers/PortfolioCalculations.sol

```diff
@@ -488,11 +495,9 @@ contract PortfolioCalculations is ExponentialNoError {
     }
 
     uint256 performanceIncrease = _currentPrice - _highWaterMark;
-    uint256 performanceFee = (performanceIncrease *
+    uint256 performanceFee = ((performanceIncrease *
       _totalSupply *
-      _feePercentage) /
-      ONE_ETH_IN_WEI /
-      TOTAL_WEIGHT;
+      _feePercentage) * ONE_ETH_IN_WEI) / TOTAL_WEIGHT;
```

Comment:
Why did you change the performance fee calculation?

Reply:
Ohkk will revert it